hw/ip/otbn/data/otbn.hjson

```diff
@@ -162,9 +179,24 @@
       hwaccess: "hwo",
       hwext: "true",
       fields: [
-        { bits: "0",
-          name: "busy",
-          desc: "OTBN is performing an operation."
+        { bits: "7:0",
+          name: "status",
+          resval: 0,
+          desc: '''
+            Indicates the current state OTBN is in.
+
+            0 (IDLE): OTBN is not performing any action.
+
+            1 (BUSY_EXECUTE): OTBN is executing software.
+
+            2 (BUSY_SEC_WIPE_DMEM): OTBN is securely wiping the data memory.
+
+            3 (BUSY_SEC_WIPE_IMEM): OTBN is securely wiping the instruction
+                memory.
+
+            255 (LOCKED): OTBN is locked as reaction to a fatal error, and
```

Comment:
Is system reset required to leave the `LOCKED` state?; and, 
Can `*_WIPE*_` be executed before reset?

Reply:
The LOCKED state is reached after a fatal alert is issued; at this point, only a OTBN reset gets us out of this state. Since we currently don't expect to do individual block resets, only a full system reset gets us out of this state.

A secure wipe is performed as part of handling a fatal error, which then also gets us into the LOCKED state. So yes, whenever we're in locked, we have securely wiped everything before.

Reply:
Sounds good thanks.